superset-frontend/packages/superset-ui-core/test/chart/clients/ChartClient.test.ts

```diff
@@ -24,17 +24,15 @@ import {
   buildQueryContext,
   QueryFormData,
   configure as configureTranslation,
-} from '@superset-ui/core/src';
-
-import {
   ChartClient,
   getChartBuildQueryRegistry,
   getChartMetadataRegistry,
   ChartMetadata,
-} from '@superset-ui/core/src/chart';
-import { SliceIdAndOrFormData } from '@superset-ui/core/src/chart/clients/ChartClient';
+} from '@superset-ui/core';
+
 import { LOGIN_GLOB } from '../fixtures/constants';
 import { sankeyFormData } from '../fixtures/formData';
+import { SliceIdAndOrFormData } from '../../../src/chart/clients/ChartClient';
```

Comment:
Since the packages are self-contained, is there a way to set something in each `package.json` or globally to make `packages/<package-name>` as the root for each package and avoid the `../.../...` imports?

Reply:
IMO, have to keep a simple and straightforward approach to link modules. We have to consider multiple runtime to alias these module
- Eslint
- Jest
- Webpack

This pattern only keep 1 principle "public variable name import from top level"